docs/devel/hotswap.md

```diff
@@ -34,11 +34,11 @@ scripts/generate-deployment-dev karydia/karydia eu.gcr.io/gardener-project/karyd
 
 ## <a name="getting-started"></a> Getting started
 
-Follow the steps of [installing karydia](../install.md)
+Follow the steps of [installing karydia](../install/README.md)
 
 Switch from karydia prod to dev Kubernetes deployment:
 ```
-kubectl delete -f manifests/deployment.yml
+kubectl delete -f install/charts/templates/deployment.yml
```

Comment:
Probably fails due to placeholders in deployment.yml. The description should reference the concrete deployment name instead of the yaml file.

Reply:
@SAPJonasZengerle